test/test_positioning.rb

```diff
@@ -605,11 +591,14 @@ def test_that_position_columns_has_default_column
   end
 
   def test_that_position_columns_does_not_need_a_scope
-    assert_equal({position: {scope_columns: [], scope_associations: []}}, Category.positioning_columns)
+    assert_equal({position: {scope_columns: [], scope_associations: []}}, Product.positioning_columns)
   end
 
   def test_that_position_columns_can_have_multiple_entries
-    assert_equal({position: {scope_columns: ["list_id"], scope_associations: [:list]}, category_position: {scope_columns: ["list_id", "category_id"], scope_associations: [:list, :category]}}, CategorisedItem.positioning_columns)
+    assert_equal({
```

Comment:
Can you fix up the line breaks here so it's formatted as it was before?

Reply:
Done.